sync/agent/process.go

```diff
@@ -103,6 +103,13 @@ func (s *Server) CreateProcess(rw http.ResponseWriter, req *http.Request) error
 
 	p.ExitCode = -2
 	go func() {
+		// set aws credential
```

Comment:
Put in `launch` for backup related processes. Other commands don't need those env variables.

Also enforce the parameter validity check there as well, since it's API.